client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -19,11 +56,26 @@ const getTitanFeatures = () => {
 	];
 };
 
-export const professionalEmailCard: ProviderCard = {
+const professionalEmailFormattedPrice = (
+	formattedPriceClassName: string,
+	cost: number,
+	currencyCode: string
+) => {
+	return translate( '{{price/}} /mailbox', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{price/}} /mailbox /month' )` *ES Score: 8*

Reply:
I think we should indeed mention the billing interval 🙂.

Reply:
I agree with Dale, it should be actually dynamic, depending on the monthly/annual selector. That would be done with the next component too.